docs/streams/developer-guide.html

```diff
@@ -2309,6 +2309,25 @@ <h4><a id="streams_developer-guide_interactive-queries_local-window-stores" href
             System.out.println("Count of 'world' @ time " + windowTimestamp + " is " + next.value);
           }
           iterator.close();
+
```

Comment:
Seems the code above is indented with 5 spaces instead of 4. Would you mind cleaning this up?